src/tests/ftest/container/root_container_test.py

```diff
@@ -0,0 +1,273 @@
+        self.device = "scm"
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super(RootContainerTest, self).setUp()
+        self.dfuse = None
+        self.dfuse_hosts = None
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse:
+                self.dfuse.stop()
+        finally:
+            # Stop the servers and agents
+            super(RootContainerTest, self).tearDown()
+
+    def _create_pool(self):
+        """Create a TestPool object to use with ior.
+        """
+        # Get the pool params
+        pool = TestPool(
+            self.context, dmg_command=self.get_dmg_command())
+        pool.get_params(self)
+        # Create a pool
+        pool.create()
+        self.pool.append(pool)
+        return pool
+
+    def _create_cont(self, pool, path=None):
+        """Create a TestContainer object to be used to create container.
+
+           Args:
+               pool (TestPool): pool object
+               path (str): Unified namespace path for container
+        """
+        # Get container params
+        container = TestContainer(pool, daos_command=DaosCommand(self.bin))
+        container.get_params(self)
+        if path is not None:
+            container.path.update(path)
+        # create container
+        container.create()
+        self.container.append(container)
+        return container
+
+    def _start_dfuse(self, pool, container):
+        """Create a DfuseCommand object to start dfuse.
+
+           Args:
+               container: Container to mount dfuse
+        """
+
+        # Get Dfuse params
+        self.dfuse = Dfuse(self.dfuse_hosts, self.tmp)
+        self.dfuse.get_params(self)
+
+        # update dfuse params
+        self.dfuse.set_dfuse_params(pool)
+        self.dfuse.set_dfuse_cont_param(container)
+        self.dfuse.set_dfuse_exports(self.server_managers[0], self.client_log)
+
+        try:
+            # start dfuse
+            self.dfuse.run()
+        except CommandFailure as error:
+            self.log.error("Dfuse command %s failed on hosts %s",
+                           str(self.dfuse),
+                           self.dfuse.hosts,
+                           exc_info=error)
+            self.fail("Test was expected to pass but it failed.\n")
+
+    def test_rootcontainer(self):
+        """Jira ID: DAOS-3782.
+
+        Test Description:
+            Purpose of this test is to try and create a container and
+            mount it over dfuse and use it as a root container and create
+            subcontainers underneath it and insert several files and see
+            if they can be accessed using ls and cd. Verify the pool size
+            reflects the space occupied by container. Try to remove the
+            files and containers and see the space is reclaimed.
+            Test the above procedure with 100 sub containers.
```

Comment:
I'm seeing the console output is running with 5 pools and 5 containers each pool, as specified in the yaml.
Did you mean to test this with (1) 100 containers and (2) 50 containers across 5 pools ?